src/SpectatorView.Unity/Assets/SpatialAlignment/README.md

```diff
@@ -1,5 +1,9 @@
 # Spatial Alignment
 
+Spatial Alignment component provides abstractions for localization of MR content within the physical world. This also incldues abstractions and implementations for the process of exchanging localization information between devivces.
```

Comment:
subjective nit: This could be "The Spatial Alignment component" or "Spatial Alignment".